### PR TITLE
A program for printing the sub sequences of a given string using recursion

### PR DESCRIPTION
**Pull request by itsnitish22**
Used recursion to print all the sub sequences of a given string.

### CODE REVIEW

File: DSA/Cpp/Recursion_and_Backtracking/TowerOfHanoi.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    TOH(n - 1, a, c, b);
+    cout << n << " [ " << a << " -> " << b << " ]" << endl;
+    TOH(n - 1, c, b, a);
+}
+
+int main()
+{
+    TOH(3, 'A', 'B', 'C');
+    return 0;
+}
```
Comment (achitJ) [inline DSA/Cpp/Recursion_and_Backtracking/TowerOfHanoi.cpp:17]: Do not add a code that you didn't mention in the issue or the PR in the first place.

**Review by achitJ — CHANGES_REQUESTED** [review]

**Comment by itsnitish22** [comment]
Raised the issue. Now you can go through the code and suggest any changes if required.

**Review by achitJ — APPROVED** [review]
LGTM

**Comment by itsnitish22** [comment]
Any updates?

**Comment by Bhavye-Malhotra** [comment]
Lgtm too

**Comment by itsnitish22** [comment]
Just saw that your repository got excluded for hacktoberfest. Why is it so?

**Pull request merged**